uportal-war/src/main/resources/properties/i18n/Messages.properties

```diff
@@ -391,6 +391,7 @@ most.frequently.added=Most Frequently Added
 move=Move
 move.this.column=Move this column
 move.this.portlet=Move this portlet
+move.this.portlet.error=This change was disallowed by the server.  The portlet will return to its original position.  Reload the page now?
```

Comment:
The goal is that an invalid move would not be allowed in the first place and the markup generated has classes to help the javascript in the UI know this.  However since the logic is implemented in two places they can (and obviously from this PR have) been out of sync.  This message captures when the UI thinks it can do something, but the back end fails the change.  Drew had mentioned that the current implementation does the move then calls the back end.  You have a point that it would be nice if that order of operations was reversed if possible.